### DISCUSSION TITLE
already exists in database

### DISCUSSION THREAD

**Discussion by zhouLife** [YOURLS/YOURLS#3099] [Q&A] [answered]
How to set multiple  Short URL  to jump to one Original URL

![chrome_EQeJ5OGuf9999](https://user-images.githubusercontent.com/47948826/139115728-0b8b2c33-7a24-4cd3-9d52-c7dee07c103c.jpg)

**Comment by dgw** [comment, marked as answer]
This was just answered last week in #3096.

**Reply by zhouLife** [reply]
Thanks, I found it
**/user/config.php**
`define( 'YOURLS_UNIQUE_URLS', false );`
